docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -8,101 +8,126 @@ ms.custom: "seodec18"
+  ```dotnetcli
+  dotnet new sln -o unit-testing-using-dotnet-test
+  ```
+
+  The [`dotnet new sln`](../tools/dotnet-new.md) command creates a new solution in the *unit-testing-using-dotnet-test* directory.
+* Change directories to the *unit-testing-using-dotnet-test* folder.
+* Run the following command:
+
+  ```dotnetcli
+  dotnet new classlib -o PrimeService
+  ```
+
+   The [`dotnet new classlib`](../tools/dotnet-new.md) command creates a new class library project  in the *PrimeService* folder. The new class library will contain the code to be tested.
+* Rename *Class1.cs* to *PrimeService.cs*.
+* Replace the code in *PrimeService.cs* with the following code:
+  
+  ```csharp
+    using System;
+
+    namespace Prime.Services
     {
-        public bool IsPrime(int candidate)
+        public class PrimeService
         {
-            throw new NotImplementedException("Please create a test first.");
+            public bool IsPrime(int candidate)
+            {
+                throw new NotImplementedException("Not implemented.");
+            }
         }
     }
-}
-```
+  ```
 
-Change the directory back to the *unit-testing-using-dotnet-test* directory.
+* The preceding code:
+  * Throws a <xref:System.NotImplementedException> with a message indicating it's not implemented.
+  * Is updated later in the tutorial.
 
-Run the [dotnet sln](../tools/dotnet-sln.md) command to add the class library project to the solution:
+<!-- preceding code shows an english bias. Message makes no sense outside english -->
 
-```dotnetcli
-dotnet sln add ./PrimeService/PrimeService.csproj
-```
+* In the *unit-testing-using-dotnet-test* directory, run the following command to add the class library project to the solution:
 
-## Creating the test project
+  ```dotnetcli
+  dotnet sln add ./PrimeService/PrimeService.csproj
+  ```
 
-Next, create the *PrimeService.Tests* directory. The following outline shows the directory structure:
+* Create the *PrimeService.Tests* project by running the following command:
 
-```
-/unit-testing-using-dotnet-test
-    unit-testing-using-dotnet-test.sln
-    /PrimeService
-        Source Files
-        PrimeService.csproj
-    /PrimeService.Tests
-```
+  ```dotnetcli
+  dotnet new xunit -o PrimeService.Tests
+  ```
 
-Make the *PrimeService.Tests* directory the current directory and create a new project using [`dotnet new xunit`](../tools/dotnet-new.md). This command creates a test project that uses [xUnit](https://xunit.github.io/) as the test library. The generated template configures the test runner in the *PrimeServiceTests.csproj* file similar to the following code:
+* The preceding command:
+  * Creates the *PrimeService.Tests* project in the *PrimeService.Tests* directory. The test project uses [xUnit](https://xunit.github.io/) as the test library.
+  * Configures the test runner by adding the following `<PackageReference />`elements to the project file:
+     * "Microsoft.NET.Test.Sdk"
+     * "xunit"
```

Comment:
```suggestion
    * "xunit"
```

Reply:
No, this is correct. It needs to be the next level indented.

Reply:
@Rick-Anderson, I removed only one space. so it's indented 4 spaces instead of 5. The linter was showing an error because of the 5 spaces.